app/services/import_service.rb

```diff
@@ -0,0 +1,49 @@
+class ImportService
+  def initialize(storage_service)
+    @storage_service = storage_service
+  end
+
+  def update_organisations(folder)
```

Comment:
Scenario:

We import Organisations 1 & 2 and they start using the new service. Organisation 1 updates their phone number in the new service. Organisation 3 is added to the S3 bucket and we run `update_organisations` again. 

Does Organisation 1's phone number get reset back to it's value in the old system? I think that's what would happen with the upsert right?

Reply:
Correct, we assumed that we would have potential updates via the rake task. Happy to prevent updates instead if you think this is our normal use case.

Reply:
I think we need to prevent updates because the assumption is that once we've imported them into the new system they stop using the old system and use the new one exclusively (though we probably need to confirm this will be enforced by softwire).

Alternatively we could use the last edit date and take the most recent?

Reply:
Decided to prevent duplicates since we don't have a date in the XML (we would have to trust the modification date on the file instead).